src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -362,6 +372,17 @@ data class DeliverablePayload(
   )
 }
 
+data class ImportDeliverableProblemElement(
+    val row: Int,
+    val problem: String,
+) : SuccessResponsePayload
```

Comment:
I'm still unclear on why this needs to be a `SuccessResponsePayload`. It's never returned as a top-level response, so effectively this just adds a hardwired `"status": "ok"` to every element of the `problems` array.